<?xml version="1.0"?>
<launch>

    <arg name="port" default="/dev/ttyUSB0"/>

    <rosparam file="$(find clober_serial)/config/clober.yaml" command="load"/>

    <node pkg="clober_serial" type="clober_serial_node" name="clober_serial_node" output="screen">
        <param name="port" value="$(arg port)"/>
    </node>

    

</launch>


